concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1323,7 +1323,7 @@ jobs:
                   mkdir -p /var/vcap/jobs/acceptance-tests/bin/ /var/vcap/packages/acceptance-tests/src/github.com/cloudfoundry
                   ln -snf $(pwd)/test-config/config.json /var/vcap/jobs/acceptance-tests/bin/config.json
                   ln -snf $(pwd)/test-config/run /var/vcap/jobs/acceptance-tests/bin/run
-                  ln -snf /usr/local/go /var/vcap/packages/golang1.4
+                  ln -snf /usr/local/go /var/vcap/packages/golang1.6
                   sed -i 's/bits=@"%s"/bits=@%s/' cf-release/src/github.com/cloudfoundry/cf-acceptance-tests/helpers/v3_helpers/v3.go
```

Comment:
Incidentally, we can remove L1327 (a revert of 4865dc54ab9199779bad4819f26861c31c4a5b3f), because it's now fixed upstream:
```
➜  cf-release git:(master) git submodule-contains src/github.com/cloudfoundry/cf-acceptance-tests cfacc2d8c7c04c3e540622a53d747ac6ed2ac332
d1cdf1b59d08ca30ee39a593e2512231db3e949e
➜  cf-release git:(master) git tag --contains d1cdf1b59d08ca30ee39a593e2512231db3e949e
v232
v233
v234
v235
v236
```

It might be worth doing that now so that we don't have cruft for tests that we're not running.